main.event_subscriptions.tf

```diff
@@ -0,0 +1,54 @@
+resource "azapi_resource" "event_subscriptions" {
```

Comment:
You should set response_export_values to an empty [] if you don't need any outputs